src/graph/client/graphClient.ts

```diff
@@ -13,8 +13,8 @@
 declare let d3: any;
 
 const animationStepMs = 50;
-const graphWidth = 1200, graphHeight = 700;
 const defaultQuery = "g.V()";
+let graphWidth = 1200, graphHeight = 700;
```

Comment:
Should this be calculated immediately rather than using arbitrary values?  I don't want it to flicker.  Do we even need these variables?

Reply:
I don't know if we need this or not.  I kept them as a default in case resultsBackground didn't render by the time I set the width/height of the force window.  I didn't notice any flickering.

I'm fine with removing them though.

Reply:
1) hard-code linkDistance, so you can get rid of the file-scoped variables
2) Move resutsBackground to htmlElements
3) No need for try - if there is no resultsBackground element, it should simply cause an error - being defensive without a reason can hide issues with the code, so you shouldn't just use it everywhere.